src/services/ai_service.py

```diff
@@ -1,9 +1,11 @@
 from typing import List, Dict
 from unidiff import Hunk, PatchedFile
+
 from ..core.config import Config
 from ..core.models import PRDetails
 from .llms.gemini import GeminiService
 from .llms.openai import OpenAIService
+from .llms.anthropic import AnthropicService
```

Comment:
Lỗi: Thông báo lỗi `Failed to initialize OpenAI service: {e}` không chính xác khi gặp lỗi trong quá trình khởi tạo `AnthropicService`.  Nên sửa lại thông báo lỗi cho phù hợp với dịch vụ đang khởi tạo. Ví dụ: `Failed to initialize Anthropic service: {e}`.